mmv1/products/oracledatabase/CloudVmCluster.yaml

```diff
@@ -76,6 +82,12 @@ examples:
     test_vars_overrides:
       'deletion_protection': 'false'
       'project': '"oci-terraform-testing"'
+      # ofake- prefix is needed to create a dummy resource for testing purposes only
+      # See: https://github.com/hashicorp/terraform-provider-google/issues/19983#issuecomment-2516403770
+      # As a result these resources are not sweepable
+      # See: https://github.com/hashicorp/terraform-provider-google/issues/20599
+      cloud_vm_cluster_id: 'fmt.Sprintf("ofake-tf-test-full-%s", acctest.RandString(t, 10))'
+      cloud_exadata_infrastructure_id: 'fmt.Sprintf("ofake-tf-test-full-%s", acctest.RandString(t, 10))'
```

Comment:
cloud_exadata_infrastructure_id: 'fmt.Sprintf("ofake-tf-test-exadata-for-vmcluster-full-%s", acctest.RandString(t, 10))'